pyartcd/tests/pipelines/test_promote.py

```
@@ -2160,3 +2160,89 @@ async def test_push_manifest_list_uses_registry_config(
+        runtime.new_slack_client.return_value.say.return_value = {'message': {'ts': ''}}
+        runtime.new_slack_client.return_value.bind_channel = MagicMock()
+
+        pipeline = await PromotePipeline.create(
+            runtime, group="openshift-4.10", assembly="4.10.99", signing_env="prod", skip_sigstore=True
+        )
+        pipeline.check_blocker_bugs = AsyncMock()
+        pipeline.change_advisory_state_qe = AsyncMock()
+        pipeline.get_advisory_info = AsyncMock(return_value={"id": 2, "errata_id": 2, "status": "QE"})
+
+        # Pipeline will fail later (no live ID), but blocker check should already have been skipped
+        with self.assertRaises(Exception):
```

Ruff (0.15.12)

[warning] Do not assert blind exception: `Exception`

(B017)